public/content/documentation/native/controls/checkbox.md

```diff
@@ -0,0 +1,295 @@
+## General Notes
+
+How to test a checkbox
+
+## Videos
+
+### iOS VoiceOver
+<video controls>
+  <source src="media/video/native/checkbox/checkboxVoiceOver.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+### Andriod Talkback
+<video controls>
+  <source src="media/video/native/checkbox/checkboxTalkback.mp4" type="video/webm">
+  Your browser does not support the video tag.
+</video>
+
+## Condensed
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a checkbox
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys or ctl+tab: Focus visibly moves to the checkbox
+   - Spacebar: Activates on iOS and Android
+   - Enter: Activates on Android
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its name, role, state
+   - Doubletap: Checkbox toggles between checked and unchecked states
+
+3. Listen to screenreader output on all devices
+
+   - Name: Name describes the purpose of the control and matches the visible label
+   - Role: Identifies itself as a checkbox in Android and a Button or checkbox in iOS
+   - Group: Visible label can be grouped with the checkbox in a single swipe
+   - State: Expresses its state (disabled/dimmed, checked, not checked, selected, unselected)
+
+4. Test device settings
+
+   - Text resize: Text label can resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/checkbox)
+
+## Gherkin
+
+### #a11y - Native Accessibility Acceptance Criteria
+
+How to test a checkbox
+
+GIVEN THAT I am on a screen with a checkbox
+
+1. Scenario: Test keyboard actions
+
+   - WHEN I press the "TAB", "ARROW KEYS", or "CTRL+TAB" keys 
+      - THEN the focus should visibly move to the checkbox 
+   - WHEN I press the "SPACEBAR" key 
+      - THEN the checkbox should be activated on iOS and Android 
+   - WHEN I press the "ENTER" key 
+      - THEN the checkbox should be activated on Android
+
+2. Scenario: Test mobile screen reader gestures
+
+   - WHEN I swipe to navigate to the checkbox 
+      - THEN the focus should move to the checkbox 
+         - AND the checkbox's name, role, and state should be expressed 
+   - WHEN I double-tap the checkbox 
+      - THEN the checkbox should toggle between CHECKED and NOT CHECKED states 
+
+3. Test screen reader output on all devices
```

Comment:
Same as above. Will we be adding "Scenario" to this one or removing "Scenario" from the other three?

Reply:
I should add it!